tajo-core/src/main/java/org/apache/tajo/master/exec/QueryExecutor.java

```diff
@@ -327,6 +328,11 @@ public void execNonFromQuery(QueryContext queryContext, Session session, String
     LogicalRootNode rootNode = plan.getRootBlock().getRoot();
 
     EvalContext evalContext = new EvalContext();
+
+    //Non From query should be session's time zone. e,g, select to_char(now(), 'yyyy-MM-dd')
+    String timezoneId = queryContext.get(SessionVars.TIMEZONE);
```

Comment:
SessionVars.TIMEZONE is designed to be used only in client and master, and QueryVars are disigned to use some variables during query execution. How about adding a new configuration key for timezone to QueryVars?